WalletWasabi/Logging/Logger.cs

```diff
@@ -222,18 +223,18 @@ private static void Log(LogLevel level, string message, string category, int add
 						}
 					}
 
-					if (FileEntryEncryptionPassword != null)
+					if (FileEntryEncryptionPassword is null)
+					{
+						File.AppendAllText(FilePath, finalFileMessage);
+					}
+					else
 					{
 						// take the separator down and add a comma (not base64)
 						var replacedSeparatorWithCommaMessage = finalFileMessage.Substring(0, finalFileMessage.Length - EntrySeparator.Length);
 						var encryptedLogMessage = StringCipher.Encrypt(replacedSeparatorWithCommaMessage, FileEntryEncryptionPassword) + ',';
 
 						File.AppendAllText(FilePath, encryptedLogMessage);
 					}
-					else
-					{
-						File.AppendAllText(FilePath, finalFileMessage);
-					}
 				}
```

Comment:
This does, but since we're removing encryption this won't either.

Reply:
I don't know about the encryption part. I simply changed the code to be easily readable.